Breadcrumbs.py

```diff
@@ -1,12 +1,18 @@
 # -*- coding: utf-8 -*-
-import sublime, sublime_plugin
 import math
+import re
+import html
+
+import sublime
+import sublime_plugin
 
 try:
   xrange
 except NameError:
   xrange = range
 
+settings = sublime.load_settings('Breadcrumbs.sublime-settings')
```

Comment:
Could the idea to make it scope-based, instead of file-type-based, be used to solve situations in which a file has multiple sections written in different languages. Say, a *.jsx file with [hyperscript](https://jasonformat.com/wtf-is-jsx/) parts, or an *.html file with inline javascript and css parts? That would be indeed superior to forcing the same rules for the whole file. Does ST give access to this kind of metadata about the current selection?